Equilobe.DailyReport.Models/ReportFrame/IndividualDraftInfo.cs

```diff
@@ -12,7 +12,7 @@ public class IndividualDraftInfo
         public string Username { get; set; }
         public string UniqueUserKey { get; set; }
         public DateTime? LastConfirmationDate { get; set; }
-        public DateTime ReportDate { get; set; }
+        public string ReportDate { get; set; }
```

Comment:
Aici poate ar trebui un comment cu formatul. 

Acest ReportDate se salveaza si in baza de date?